src/table/src/TableRow.js

```diff
@@ -128,11 +127,6 @@ TableRow.propTypes = {
    */
   isHighlighted: PropTypes.bool,
 
-  /**
-   * The intent of the alert.
-   */
-  intent: PropTypes.oneOf(['none', 'success', 'warning', 'danger']),
```

Comment:
I think we are considering keeping `intent` around just to minimize API changes, especially since we can support it pretty easily with the theming solution we have now.